submissions/YuliiaHordiichuk/memory-game/index.js

```diff
@@ -0,0 +1,91 @@
+class Game {
+    constructor() {
+        this.cardsContainer = document.querySelector('.cards'); 
+        this.cardsBack = ['cat', 'dog', 'elephant', 'fox', 'jaguar', 'zebra'];
+        this.matchCards = 0; 
+        this.winMatchCards = 6; 
+        this.oneMatchCards = 2; 
+        this.openCards = []; 
+        this.delayToCloseCards = 400; 
+        this.delayToHideCards = 400; 
+        this.delayToAlertWin = 600;
+
+        this.fillCards();
+        this.addClickHendler();
+    }
+
+    fillCards () {
+        this.cardsContainer.innerHTML = this.createCards();
+    }
+
+    mixCards () {
+        return this.cardsBack.concat(this.cardsBack).sort(() =>  0.5 - Math.random());  
+    }
+
+    createCards () {
+        return this.mixCards().map(card => {
+            return `
+            <li class='card'>
+                <button class='card__btn' data-id='${card}'>
```

Comment:
Why do you need a button here?

Reply:
I use the button in this case to provide keyboard accessibility